metaci/release/utils.py

```diff
@@ -67,6 +67,52 @@ def send_release_webhook(project_config, release):
         "package_name": project_config.project__package__name,
         "version": project_config.get_version_for_tag(tag),
         "release_url": f"{release.repo.url}/releases/tag/{urllib.parse.quote(tag)}",
+        "steps": [
+            {
+                "description": "release_deploy",
+                "start_time": release.implementation_steps.get(
+                    plan__role="release_deploy"
+                ).start_time.isoformat(),
+                "end_time": release.implementation_steps.get(
+                    plan__role="release_deploy"
+                ).stop_time.isoformat(),
+                "implementation_steps": "release_deploy",
+                "configuration_item": config_item
+            },
+            {
+                "description": "release",
+                "start_time": release.implementation_steps.get(
+                    plan__role="release"
+                ).start_time.isoformat(),
+                "end_time": release.implementation_steps.get(
+                    plan__role="release"
+                ).stop_time.isoformat(),
+                "implementation_steps": "release",
+                "configuration_item": config_item
+            },
+            {
+                "description": "push_sandbox",
+                "start_time": release.implementation_steps.get(
+                    plan__role="push_sandbox"
+                ).start_time.isoformat(),
+                "end_time": release.implementation_steps.get(
+                    plan__role="push_sandbox"
+                ).stop_time.isoformat(),
+                "implementation_steps": "push_sandbox",
+                "configuration_item": config_item
+            },
+            {
+                "description": "push_production",
+                "start_time": release.implementation_steps.get(
+                    plan__role="push_production"
+                ).start_time.isoformat(),
+                "end_time": release.implementation_steps.get(
+                    plan__role="push_production"
+                ).stop_time.isoformat(),
+                "implementation_steps": "push_production",
+                "configuration_item": config_item
+            },
+        ],
```

Comment:
We should:
1. Only add these steps if config_item has a value.
2. Instead of a hardcoded list of implementation steps, we should create a list from release.implementation_steps.all() and loop over it to add each of those. That way it's possible to customize the steps for a particular release if necessary.